.github/ISSUE_TEMPLATE.md

```diff
@@ -16,7 +16,8 @@
 [[PLACE YOUR OUTPUT HERE]]
 ```
 
-`/var/tmp $ java -Duser.language=en -Duser.country=US -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java`:
+`/var/tmp $ java -Duser.language=en -Duser.country=US
```

Comment:
doesn't this need something like `\` to say it is a line continuation?

Reply:
hmm, what about such template ?:
```
/var/tmp $ javac YOUR_FILE.java
#[[PLACE YOUR OUTPUT HERE]]

/var/tmp $ cat YOUR_FILE.java
#[[PLACE YOU OUTPUT HERE]]

/var/tmp $ cat config.xml
#[[PLACE YOUR OUTPUT HERE]]

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-X.XX-all.jar -c config.xml YOUR_FILE.java
#[[PLACE YOUR OUTPUT HERE]]
```

![image](https://user-images.githubusercontent.com/812984/40196976-ef899da2-59c6-11e8-8928-803ad095eb31.png)

Reply:
I am fine with either way you want to go.

Reply:
updated